### PR TITLE
[TG-58] Save/Edit Wizard form

### PR DESCRIPTION
**Pull request by antonSavk**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/31" target="_blank">ReviewNinja

File: src/API/scenarios/scenario.test.js [hunk 1]
```diff
@@ -19,7 +19,15 @@ describe('## scenario APIs', () => {
             _id: mongoose.Types.ObjectId('000000000000000000000000'),
             name: 'Greeting script',
             description: 'will greet you in a brand new world',
-            body: 'console.log("Hello Scripto World!");'
+            body: 'console.log("Hello Scripto World!");',
+            active: true,
+            isConvertible: false,
+            jsCode: '',
```
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fix
initially `body` was intended to hold scripts
need to remove one of that

File: src/API/scenarios/scenario.test.js [hunk 2]
```diff
@@ -19,7 +19,15 @@ describe('## scenario APIs', () => {
...
+            generatedJson: null,
+            sourceType: '',
+            actions: [],
+            conditions: [],
+            logicalOperator: ''
```
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fix
why do we need `generatedJson` aside with `actions`, `conditions`, `logicalOperator`
(not sure what `sourceType` holds)

File: src/API/scenarios/scenario.test.js [hunk 3]
```diff
@@ -79,10 +87,11 @@ describe('## scenario APIs', () => {
         });
     });
 
-    describe('# POST /api/scenarios', () => {
+    xdescribe('# POST /api/scenarios', () => {
```
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fix

Comment (antonSavk) [inline src/API/scenarios/scenario.test.js]: @kosich xyix

File: src/API/scenarios/scenario.test.js (hunk 2 above)
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fixed

File: src/API/scenarios/scenario.test.js (hunk 3 above)
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fixed

File: src/API/scenarios/scenario.test.js (hunk 1 above)
Comment (kosich) [inline src/API/scenarios/scenario.test.js]: !fixed

**Pull request merged**